GetIntoTeachingApiTests/Models/CandidateQualificationTests.cs

```diff
@@ -11,11 +11,16 @@ public class CandidateQualificationTests
         [Fact]
         public void Constructor_WithEntity_MapsCorrectly()
         {
-            var entity = new Entity();
-            entity.Id = Guid.NewGuid();
-            entity.Attributes["dfe_category"] = new OptionSetValue { Value = 1 };
-            entity.Attributes["dfe_type"] = new OptionSetValue { Value = 2 };
-            entity.Attributes["dfe_degreestatus"] = new OptionSetValue { Value = 3 };
+            var entity = new Entity
+            {
+                Id = Guid.NewGuid(),
+                Attributes =
+                {
+                    ["dfe_category"] = new OptionSetValue {Value = 1},
+                    ["dfe_type"] = new OptionSetValue {Value = 2},
+                    ["dfe_degreestatus"] = new OptionSetValue {Value = 3}
+                }
+            };

```

Comment:
I would avoid inline construction like this ... the debugger cannot drop into this, it sees it as one line

Reply:
again, ill not comment on each instance